src/redux/actions/concepts/dictionaryConcepts.js

```diff
@@ -108,18 +108,18 @@ export const fetchDictionaryConcepts = (
   page = 1,
 ) => async (dispatch, getState) => {
   dispatch(isFetching(true));
-  let url = `${conceptType}/${conceptOwner}/collections/${conceptName}/concepts/?q=${query}&limit=${limit}&page=${page}&verbose=true`;
+  let url = `${conceptType}/${conceptOwner}/sources/${conceptName}/concepts/?q=${query}&limit=${limit}&page=${page}&verbose=true`;
```

Comment:
Why is there a change from collections to sources?

Reply:
The change is because sources display the latest changes while collections display the initial version.

Reply:
Cool